### PR TITLE
Add delete site from Cloudflare section

### PR DESCRIPTION
**Pull request by Ekwuno**
Adding a section to address #2271.

### CODE REVIEW

File: products/pages/src/content/get-started/index.md [hunk 1]
```diff
@@ -103,6 +103,23 @@ To ensure a custom domain is added successfully, you must go through the [Adding
 
 </Aside>
 
+### Deleting a custom domain
+
+The Cloudflare dashboard has a list of services that your pages project can leverage on, such as Analytics, speed, and, SSL/TLS. When trying to remove your project from Cloudfalre you will not be able to just delete the project from the pages dashboard, if your DNS is handled by Cloudflare. You will have to delete custom domains before deleting the project from Pages dashboard.
```
Comment (deadlypants1973) [inline products/pages/src/content/get-started/index.md]:
```suggestion
If your DNS is handled by Cloudflare, you must delete any custom domain associated with your Pages project before removing your project from the **Pages** dashboard. Failure to do so may leave the DNS records active, causing your domain to point to a Pages project that no longer exists.
```

File: products/pages/src/content/get-started/index.md [hunk 2]
```diff
@@ -103,6 +103,23 @@ To ensure a custom domain is added successfully, you must go through the [Adding
 
 </Aside>
 
+### Deleting a custom domain
+
+The Cloudflare dashboard has a list of services that your pages project can leverage on, such as Analytics, speed, and, SSL/TLS. When trying to remove your project from Cloudfalre you will not be able to just delete the project from the pages dashboard, if your DNS is handled by Cloudflare. You will have to delete custom domains before deleting the project from Pages dashboard. 
+
+The dashboard for your site is reachable from the drop down menu at top right of the page.
+
```
Comment (deadlypants1973) [inline products/pages/src/content/get-started/index.md]:
```suggestion
Log in to the Cloudflare dashboard and use the **Account Home** or drop-down menu to select your website.
```

File: products/pages/src/content/get-started/index.md [hunk 3]
```diff
@@ -103,6 +103,23 @@ To ensure a custom domain is added successfully, you must go through the [Adding
...
+
+![drop down menu](./images/drop-down-menu.png)
+
+
+On the dashboard overview page you have to scroll down to the advanced action option
+and click on **remove site from Cloudflare** this will bring up a modal and if you confirm, your site will be deleted from Cloudflare.
```
Comment (deadlypants1973) [inline products/pages/src/content/get-started/index.md]:
```suggestion
After you select your website, you will be directed to **Overview** on the dashboard. On **Overview**:
1. Scroll down to **Advanced Actions**.
2. Select **Remove Site from Cloudflare**.
3. To delete your website from Cloudflare, select **Confirm**.
```

File: products/pages/src/content/get-started/index.md [hunk 4]
```diff
@@ -103,6 +103,23 @@ To ensure a custom domain is added successfully, you must go through the [Adding
...
+
+![advanced-option](./images/advanced-option.png)
+
+Now if you return to the pages dashboard and go to **settings** > **delete project**. You can now remove your project from Cloudflare pages.
```
Comment (deadlypants1973) [inline products/pages/src/content/get-started/index.md]:
```suggestion
Go back to the **Account Home** or use the drop-down menu at the top of the dashboard and select **Pages**. You can now remove your project by selecting your **Pages project** > **Settings** > **Delete project**.
```

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

File: products/pages/src/content/get-started/index.md (hunk 1 above)
Comment (kristianfreeman) [reply products/pages/src/content/get-started/index.md]: general note - make sure we use `Pages` not `pages` when opening PRs!

Comment (Ekwuno) [reply products/pages/src/content/get-started/index.md]: Got it!

Comment (BOYKHMER) [reply products/pages/src/content/get-started/index.md]: https://api.bscscan.com/api
   ?module=account
   &action=balancehistory
   &address=0x0DB011018728D1B91dDB3C77933a40B9B68C9fa7
   &blockno=2000000
   &apikey=YourApiKeyToken

**Review by kristianfreeman — APPROVED** [review]

**Comment by kristianfreeman** [comment]
@Ekwuno lgtm, merge when ready

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**